config_templates/gretel/evaluate/default.yml

```diff
@@ -2,7 +2,18 @@
 # a Synthetic Quality Score report for any two datasets
 
 schema_version: "1.0"
-
+name: "evaluate-model"
 models:
   - evaluate:
-      data_source: "__tmp__"
+      data_source: "__tmp__" # Output data source
+      ref_data:
+        "0": "__tmp__" # Original data source
```

Comment:
should these have names that match for the keys, instead of comments, or do they need to be `0` and `1`?

Reply:
wanted to use the same dict structure 
<img width="1031" alt="image" src="https://github.com/user-attachments/assets/cc4465a4-79cb-4ca9-877e-cf053716ae59">